troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+   ironic_endpoint: http://172.22.0.1:6385/v1/
+   ironic_inspector_endpoint: http://172.22.0.3:5050/v1/
+   provisioning_interface: eno1
+   provisioning_ip: 172.22.0.1/24
+   rhcos_image_url: http://[Ipv4/Ipv6-IP-Registry]/rhcos-43.81.202001142154.0-openstack.x86_64.qcow2.gz
+```
+
+Before running the installation, please make sure you have a cleaned the previous files/manifests.
+
+# Bootstrap VM related issues 
+
+After the installation is triggered, Check/verify bootstrap VM is up and running from worker node: 
+
+```
+[kni@provisioner]$ sudo virsh list
+ Id    Name                           State
+----------------------------------------------------
+ 12    kni7-xf6fq-bootstrap           running
+```
+
+If there is no sign of Bootstarp VM running on the provioner node, check if the libvirtd is running(if not, start the service) on the provisioner node. 
+
+```
+systemctl status libvirtd
+● libvirtd.service - Virtualization daemon
+   Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled; vendor preset: enabled)
+   Active: active (running) since Mon 2020-02-03 13:08:51 EST; 2 weeks 0 days ago
+     Docs: man:libvirtd(8)
+```
+
+If re-running the deployment, please make sure to delete all the older VM including its volume as shown below:
+
+```
+[kni@provisioner]$ sudo virsh pool-list
+ Name                 State      Autostart 
+-------------------------------------------
+ default              active     yes       
+
+[kni@provisioner]$ sudo virsh vol-list default 
+ Name                 Path                                    
+------------------------------------------------------------------------------
+ kni7-xf6fq-bootstrap /var/lib/libvirt/images/kni7-xf6fq-bootstrap
+ kni7-xf6fq-bootstrap.ign /var/lib/libvirt/images/kni7-xf6fq-bootstrap.ign
+
+[kni@provisioner]$ sudo virsh vol-delete --vol /var/lib/libvirt/images/kni7-xf6fq-bootstrap 
+Vol /var/lib/libvirt/images/kni7-xf6fq-bootstrap deleted
+
+[kni@provisioner]$ sudo virsh vol-delete --vol /var/lib/libvirt/images/kni7-xf6fq-bootstrap.ign 
+Vol /var/lib/libvirt/images/kni7-xf6fq-bootstrap.ign deleted
+```
+
+Once this is done delete the VM including the storage pool:
+
+```
+[kni@provisioner]$ sudo virsh destroy kni7-xf6fq-bootstrap
+Domain kni7-xf6fq-bootstrap destroyed
+
+[kni@provisioner]$ sudo virsh undefine kni7-xf6fq-bootstrap
+Domain kni7-xf6fq-bootstrap has been undefined
+```
+Or simply make use of 
+
+```
+BOOTSTRAP=$(virsh list --all | grep "$CLUSTER.*bootstrap" | tail -1 | awk '{print $2}')
+virsh destroy $BOOTSTRAP
+virsh undefine $BOOTSTRAP
+virsh vol-delete $BOOTSTRAP default
+virsh vol-delete $BOOTSTRAP.ign default
+```
+
+* You will get the IP of this VM by issuing `sudo virsh net-dhcp-leases` usually always the private IP of this VM is `172.22.0.2` , you can login to this VM using the private SSH key with core as its user:
+
+```
+# ssh -i ~/.ssh/id_rsa core@172.22.0.2
+```
+
+and then issuing the journalctl command to inspect the logs:
```

Comment:
What am I inspecting for? There is no concrete problem I'm trying to solve here. You are just mentioning logs (which is good) but context is important.

Reply:
Its just a way to tell a user to start checking once h logs inside this VM, journalctl generally states if there is any error in itself.